rego/Drive.rego

```diff
@@ -73,36 +119,61 @@ tests contains {
 }
 if {
     DefaultSafe := false
-    Events := utils.FilterEvents(LogEvents, "SHARING_OUTSIDE_DOMAIN", utils.TopLevelOU)
+    Events := utils.FilterEventsOU(LogEvents, "SHARING_OUTSIDE_DOMAIN", utils.TopLevelOU)
     count(Events) == 0
 }
 
 tests contains {
     "PolicyId": "GWS.DRIVEDOCS.1.2v0.1",
     "Criticality": "Should",
-    "ReportDetails": utils.ReportDetailsOUs(NonCompliantOUs1_2),
-    "ActualValue": {"NonCompliantOUs": NonCompliantOUs1_2},
+    "ReportDetails": utils.ReportDetails(NonCompliantOUs1_2, NonCompliantGroups1_2),
+    "ActualValue": {"NonCompliantOUs": NonCompliantOUs1_2,
+        "NonCompliantGroups": NonCompliantGroups1_2},
     "RequirementMet": Status,
     "NoSuchEvent": false
 }
 if {
-    Events := utils.FilterEvents(LogEvents, "SHARING_OUTSIDE_DOMAIN", utils.TopLevelOU)
+    Events := utils.FilterEventsOU(LogEvents, "SHARING_OUTSIDE_DOMAIN", utils.TopLevelOU)
     count(Events) > 0
-    Status := count(NonCompliantOUs1_2) == 0
+    Conditions := {count(NonCompliantOUs1_2) == 0, count(NonCompliantGroups1_2) == 0 }
+    Status := (false in Conditions) == false
 }
 #--
 
 # Can be combined with 1.4 since a single policy can be used to check both conditions
 #
 # Baseline GWS.DRIVEDOCS.1.3v0.1
 #--
-NonCompliantOUs1_3 contains OU if {
+
+GetFriendlyValue1_3(Value, AcceptableValues) := "External Sharing Warning is Enabled" if {
+    Value in AcceptableValues == true
+}
+else := "External Sharing Warning is Disabled"
+
+
+NonCompliantOUs1_3 contains {
+    "Name": OU,
+    "Value": concat("", [GetFriendlyValue1_3(LastEvent.NewValue, AcceptableValues)])
+    } if {
     some OU in utils.OUsWithEvents
-    Events := utils.FilterEvents(LogEvents, "SHARING_OUTSIDE_DOMAIN", OU)
+    Events := utils.FilterEventsOU(LogEvents, "SHARING_OUTSIDE_DOMAIN", OU)
+    count(Events) > 0
+    LastEvent := utils.GetLastEvent(Events)
+    AcceptableValues := {"SHARING_ALLOWED_WITH_WARNING", "SHARING_NOT_ALLOWED",
```

Comment:
See the preceding comment, the `AcceptableValues` list is missing the following values:
- TRUSTED_DOMAINS_ALLOWED_WITH_WARNING
- TRUSTED_DOMAINS_ALLOWED_WITH_WARNING_MAY_RECEIVE_FILES_FROM_ANYONE

Also the comment "Can be combined with 1.4 since a single policy can be used to check both conditions" should be deleted.